python/cudf/cudf/tests/test_csv.py

```diff
@@ -1037,6 +1033,8 @@ def test_csv_reader_prefix():
         assert column_names[col] == prefix_str + str(col)
 
 
+# Category is not yet supported from libcudf
+@pytest.mark.xfail()
```

Comment:
May want to just remove this test. I imagine we're getting rid of the built in hashing of the CSV reader?